### PR TITLE
console: update error handling for opt-out notification

### LINKED ISSUE #4785: Opt out in UI crashes with SQL error
```
{
    "statement": "update hdb_catalog.hdb_version set console_state = '{\"disablePreReleaseUpdateNotifications\":true}' where hasura_uuid='null';",
    "prepared": false,
    "error": {
        "exec_status": "FatalError",
        "hint": null,
        "message": "invalid input syntax for type uuid: \"null\"",
        "status_code": "22P02",
        "description": null
    },
    "arguments": []
}
```

image

When you get the beta-notifications at the bottom of the console.

### PR DESCRIPTION
**Pull request by kolharsam**
### Description
Sometimes, the `hasura_uuid` isn't set on the console, and this can cause bugs on the console. this commit ~checks for a hasura_uuid to be present and avoids showing bad error messages.~ removes the unecessary check for hasura_uuid to be present

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR.

### Affected components
- [x] Console

#### Breaking changes

- [x] No Breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - server: fix importing of allow list query from metadata (fix #4687)
 - server: flush log buffer during shutdown (#4800)
 - server: fix edge case with printing logs on startup failure (fix #4772)
+- console: show internal error notification if hasura_uuid is absent on client (#4785)
```
Comment (tirumaraiselvan) [inline CHANGELOG.md:72]: I would say let's not have any changelog for this.

File: console/src/telemetry/Actions.js [hunk 2]
```diff
@@ -29,6 +29,14 @@ const setConsoleOptsInDB = (opts, successCb, errorCb) => (
     ...opts,
   };
 
+  if (!hasura_uuid) {
+    showErrorNotification(
```
Comment (beerose) [inline console/src/telemetry/Actions.js:33]: This function needs to be dispatched. Could you wrap it in `dispatch`?

File: CHANGELOG.md (hunk 1 above)
Comment (beerose) [inline CHANGELOG.md:72]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -291,7 +291,7 @@ For example, see [here](https://hasura.io/docs/1.0/graphql/manual/api-reference/
 - console: prevent trailing spaces while creating new role (close #3871) (#4497)
 - docs: add API docs for using environment variables as webhook urls in event triggers
 - server: fix recreating action's permissions (close #4377)
-- server: make the graceful shutdown logic customizable (graceful shutdown on the SIGTERM signal continues to be the default) 
+- server: make the graceful shutdown logic customizable (graceful shutdown on the SIGTERM signal continues to be the default)
```
Comment (beerose) [inline CHANGELOG.md:294]:
```suggestion
```

**Pull request merged**